.codecov.yml

```diff
@@ -35,17 +35,9 @@ coverage:
 
   # Learn more at https://docs.codecov.io/docs/commit-status
   status:
-    project:
-      default:
-        informational: true
-        # Informational doesn't work with project yet
-        threshold: 0.1
     patch:
       default:
         informational: true
```

Comment:
I wonder whether we should remove the `informational` bit here. Now that we CodeCov properly find the base commit, it will only error if the patch reduces the coverage of a file, which shouldn't happen too often.

Reply:
I recommend having that discussion in another PR.

Reply:
OK -> https://github.com/dlang/dmd/pull/7712